config.json

```diff
@@ -2,5 +2,6 @@
   "CoinAPIURL": "https://min-api.cryptocompare.com/data/pricemultifull?fsyms=",
   "WizdaddyURL": "http://dev.wizdaddy.io/api/giveItToMeDaddy",
   "InvalidCommandMessage": "Invalid Command",
-  "BotLogChannelID": "493921138657722368"
+  "BotLogChannelID": "493921138657722368",
```

Comment:
I think normally we'd want to have these as env variables, but since no one else is using the project its probably just easier this way